tests/Unit/HC/abort.cpp

```diff
@@ -0,0 +1,93 @@
+
+// RUN: %hc %s -o %t.out && %t.out
+
```

Comment:
This test is expected to abort during execution but it would be counted as a test failure if it's run as is.
The proper way would be to pipe stdout and stderr into the FileCheck tool to look for the expected error message.  You'll find examples with FileCheck in hcc/tests/Unit